cloudify_scalelist/workflows.py

```diff
@@ -622,6 +625,8 @@ def _filter_node_instances(ctx, node_ids, node_instance_ids, type_names,
 
         # look more deeply, what about instance id's and properties
         for instance in node.instances:
+            if instance.state in IGNORED_STATES:
+                continue
```

Comment:
maybe it's not my place to ask, but since you asked for a review, I'll ask anyway: why should we ignore certain instance states in the scalelist plugin?